### PR TITLE
Updated ETF install file for OSGeoLive 17 to work with new OS and Java version

### PR DESCRIPTION
**Pull request by jenriquesoriano**
Updated the ETF installation file to be compatible with both the new OsGeoLive 17 and JDK 11.
Updated user identification and assignation on folders permits.
Updated Java version and check for it before installing.
Ensure the java Version is the one required in both the start and stop script.

### CODE REVIEW

**Comment by kalxas** [comment]
Hi,

The default Java in Ubuntu 24.04 is Java 21. We have downgraded to Java 17 to make transition easier.
Will ETF work with Java 17?

**Comment by DanielMartinPerezdeLeon** [comment]
Hi @kalxas ,

Unfortunately, the ETF is not compatible with any Java version above 11 (we have tested this). In the past, we made some changes to migrate it to Java 11, upgrading from version 8.

Due to this limitation, it is necessary to run the ETF with Java 11. In the script, we ensure that Java 11 is installed if it doesn't already exist, and we configure etf-start and etf-stop to use it. However, we do not change the system's default Java version.

If you have any questions, feel free to reach out to me.

Thank you!

File: bin/install_etf.sh [hunk 1]
```diff
@@ -80,11 +80,26 @@ echo "JAVA_PKG: $JAVA_PKG"
...
+# Check if openjdk-11-jdk-headless is installed
+if dpkg -l | grep -q "$JAVA_PKG"; then
+    echo "$JAVA_PKG is installed."
+else
+    echo "$JAVA_PKG is not installed. Installing now..."
+    apt-get --assume-yes install "$JAVA_PKG"
 fi
+
+# Set JAVA system version to 11
+update-alternatives --set java /usr/lib/jvm/java-11-openjdk-amd64/bin/java
```
Comment (kalxas) [inline bin/install_etf.sh:94]: This sets Java 11 system wide.

**Comment by DanielMartinPerezdeLeon** [comment]
Dear @kalxas ,

We have modified the installation file to make sure it could never change Java Version in the system, we will close this PR and create a new one with the new changes of our branch.

Thanks for reviewing and noticing the issue.

Regards,

**Comment by DanielMartinPerezdeLeon** [comment]
Dear @kalxas ,

We have added a commit in the branch to ensure in any case that the system's Java version is not changed in any scope, to make this, we have set the Java path when starting jetty.sh to the Java-11 bin path.

I have deleted the last comment because we decided it was better to do everything in a single PR to maintain the timeline.

Thank you!

Regards,

**Comment by kalxas** [comment]
Thank you

**Pull request merged**